app/res/layout/single_app_view.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:gravity="center" >
+    
+    <TextView 
+        android:id="@+id/app_name"
+        android:textSize="50sp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingTop="5sp"
+        android:textColor="#616161"
+        android:gravity="center" />
+    
+    <TextView
+        android:id="@+id/profile_warning"
+        android:textSize="20sp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/old_profile_warning"
+        android:gravity="center"
+        android:visibility="visible" />
+                    
+    <Button
+         android:id="@+id/uninstall_button"
+         android:text="@string/uninstall_app"
+         android:textSize="40sp"
+         android:layout_width="fill_parent"
+         android:layout_height="wrap_content"
+         android:onClick="rebootAlertDialog"
```

Comment:
Curious if there's a reason you prefer setting the click handler in the xml, looks like we usually set it in java.